src/lib/storage/storage_manager.hpp

```diff
@@ -104,6 +111,7 @@ class StorageManager : public Noncopyable {
   tbb::concurrent_unordered_map<std::string, std::shared_ptr<Table>> _tables{INITIAL_MAP_SIZE};
   tbb::concurrent_unordered_map<std::string, std::shared_ptr<LQPView>> _views{INITIAL_MAP_SIZE};
   tbb::concurrent_unordered_map<std::string, std::shared_ptr<PreparedPlan>> _prepared_plans{INITIAL_MAP_SIZE};
+  std::vector<const uint32_t*> _maps;
```

Comment:
When we use `MemoryRegion`s or something similar, we will store a vector of `MemoryRegions` here. Nevertheless, the naming can be more descriptive here, indicating that we store start addresses of memory regions here.

Reply:
This vector was not well thought through.
I just purged it.